live-examples/css-examples/transitions/z-index.html

```diff
@@ -1,38 +1,59 @@
-<section id="example-choice-list" class="example-choice-list" data-property="zIndex">
-<div class="example-choice">
-<pre><code class="language-css">z-index: 1;</code></pre>
-<button type="button" class="copy hidden" aria-hidden="true">
-    <span class="visually-hidden">Copy to Clipboard</span>
-</button>
-</div>
+<section id="example-choice-list" class="example-choice-list large" data-property="zIndex">
 
-<div class="example-choice">
-<pre><code class="language-css">z-index: 3;</code></pre>
-<button type="button" class="copy hidden" aria-hidden="true">
-    <span class="visually-hidden">Copy to Clipboard</span>
-</button>
-</div>
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">z-index: auto;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
 
-<div class="example-choice">
-<pre><code class="language-css">z-index: 5;</code></pre>
-<button type="button" class="copy hidden" aria-hidden="true">
-    <span class="visually-hidden">Copy to Clipboard</span>
-</button>
-</div>
+    <div class="example-choice">
+        <pre><code class="language-css">z-index: 1;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">z-index: 3;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">z-index: 3;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
```

Comment:
This choice is a duplicate of the one before (both `z-index: 3;`). I think you could just remove one of them.

Reply:
Whoops! Fixed.